README.md

```diff
@@ -488,6 +488,28 @@ and the `GitHub` theme when in the _light mode_.
 alias cat="bat --theme=\$(defaults read -globalDomain AppleInterfaceStyle &> /dev/null && echo default || echo GitHub)"
 ```
 
+### Syntax of a file
+
+You can change the default syntax highlighting of the file using the `--map-syntax` option.
```

Comment:
This would make it more difficult for ESL speakers to understand, but I feel like this can be expressed more accurately as:

> You can change the default syntax association of files by using the `--map-syntax` option.

Reply:
Actually I am an ESL speaker myself. But yeah it is easier to read.